Add changelog for 1.12.0 release (#9262)
swissspidy/web-stories-wp: readme.txt

For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).

= 1.12.0 =

> **Release Date:** October 5, 2021.

Author: ```suggestion
**Release Date:** October 6, 2021.
```

just in case

> * New feature: ability to trim videos to make them shorter.

Reviewer: ```suggestion
* New feature: video trimming tool in the editor.
```

> * New feature: [improved analytics configuration](https://wp.stories.google/docs/seo/#Adding-Analytics) with opt-in on the settings page.

Reviewer: ```suggestion
* New feature: [improved analytics configuration](https://wp.stories.google/docs/seo/#Adding-Analytics) via opt-in on the settings page.
```

* Template colors are now available in "Saved Colors" when creating a story from a template.

> * Improved management of publisher logos in the editor.

Reviewer: ```suggestion
* Easier selection of publisher logos in the editor.
```
